pkg/scheduler/apis/config/types.go

```diff
@@ -131,3 +134,32 @@ type KubeSchedulerLeaderElectionConfiguration struct {
 	// LockObjectName defines the lock object name
 	LockObjectName string
 }
+
+type Plugins struct {
+	QueueSort      []Plugin
+	PreFilter      []Plugin
+	Filter         []Plugin
+	PostFilter     []Plugin
+	Score          []ScorePlugin
+	NormalizeScore []Plugin
+	Reserve        []Plugin
+	Permit         []Plugin
+	PreBind        []Plugin
+	Bind           []Plugin
+	PostBind       []Plugin
+	UnReserve      []Plugin
+}
+
+type Plugin struct {
+	Name string
+}
+
+type ScorePlugin struct {
```

Comment:
@bsalamat we went back and forth over whether to make this a separate struct. Are you happy with this implementation? I don't recall the details of that conversation.

Reply:
I prefer to have a single type for all plugins. The framework will ignore `Weight` where it doesn't make sense to have a weight.

Reply:
Agree. Weight it doesn't seem to work in framework now.
I have unified all plugins type. PTAL.